system/lib/wasmfs/syscalls.cpp

```diff
@@ -1389,6 +1391,82 @@ int __syscall_fcntl64(int fd, int cmd, ...) {
   }
 }
 
+intptr_t _mmap_js(intptr_t addr,
```

Comment:
Why `_js`? Is there a better name we can use?

Reply:
This is overriding the otherwise-found-in-js version of this symbol.

Perhaps it could be called `_mmap_internal` instead to avoid that confusion?